test/JDBC/expected/BABEL-4926.out

```diff
@@ -0,0 +1,225 @@
+-- create a new table BABEL_4926 1
```

Comment:
1. Let's have the meaningful test filenames which reflects the purpose instead of plain Task/jira numbers, something like `TEST-SELECT_INTO`, although feel free to have even a better name.
2. Let's have test cases in upgrade format: <testname>-vu-prepare/verify/cleanup.
3. In fact, we can use the existing test files for select-into (BABEL_539-vu-*) to add new test scenarios instead of creating separate test files. See https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/1ece7f885dba6379f775ee15213a86f570d88b72

Reply:
done